rankboost.py

```diff
@@ -436,6 +436,42 @@ def prepare_dataframe(mhc, predfile, expfile, pepfile, efilt_threshold, boost):
     _log.info('Finished creating the data frame for rank boosting..')
     return all_data, stats, boost
 
+def collapse_iars (stats):
+    """
+    Collapse duplicated IARs.
+    :param stats:
+    :return:
+    """
+    for pep_name_o in stats.index:
+        try:
+            iar_to_check = stats.ix[pep_name_o]['IAR_sequence']
+        except(KeyError):
```

Comment:
would help to explain why you are seeing this KeyError and why it is ok to `continue` in a comment within the block